### PR TITLE
[kill] Lab #1

### PR DESCRIPTION
**Pull request by smthngslv**

### CODE REVIEW

**Review by DOCtorActoAntohich — APPROVED** [review]
gud :trollface:

File: app_python/Makefile [hunk 1]
```diff
@@ -0,0 +1,143 @@
+###--VARIABLES--########################################################################################################
+
+AWS_ACCOUNT=553667903818
```
Comment (ntdesmond) [inline app_python/Makefile:3]: Why not in `.env`?

File: app_python/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+You can also start service without downloading variables from AWS. Just use `make local`. This will use `dev` as tag.
+
+8. In order to start service in the server use `make STAGE`. There are two stages - `dev` and `prod` To be able to 
+specify namespace for environment variables use NAMESPACE argument:
+```shell
+make dev
+NAMESPACE=innopolis make dev
+```
+
+9. You can stop or down your services with `make` and `stop`/`down`/`down-v`. The `down-v` command will also delete all
```
Comment (ntdesmond) [inline app_python/README.md:89]: *take down? put down? grammar issue

File: app_python/config/ecr_policy.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+   "rules": [
+       {
+           "rulePriority": 1,
+           "description": "Delete images if more than 2.",
```
Comment (ntdesmond) [inline app_python/config/ecr_policy.json:5]: images... what? what is all about?

File: app_python/config/nginx/.htpasswd [hunk 4]
```diff
@@ -0,0 +1 @@
+admin:$apr1$4pAYV/Ux$jAdm3GW8A1HHiba8XCH2X.
```
Comment (ntdesmond) [inline app_python/config/nginx/.htpasswd:1]: are these credentials just an example? why not `.example` then?

**Review by ntdesmond — APPROVED** [review]
So basically it's lab 2 as well, since Docker configs are here.
Though, you'll need to add `DOCKER.md` or whatever the second lab requires

File: app_python/Makefile (hunk 1 above)
Comment (smthngslv) [reply app_python/Makefile:3]: That are env variables that never change. So we need to set them only once

File: app_python/README.md (hunk 2 above)
Comment (smthngslv) [reply app_python/README.md:89]: oh, here we go again

File: app_python/config/ecr_policy.json (hunk 3 above)
Comment (smthngslv) [reply app_python/config/ecr_policy.json:5]: basically, you tell ECR to do not keep more than two images per tag. So it saves some space. ECR  - Elastic Container Registry. Just DockerHub from AWS

File: app_python/config/nginx/.htpasswd (hunk 4 above)
Comment (smthngslv) [reply app_python/config/nginx/.htpasswd:1]: actually, there should be real creds. It's safe because they are hashed

**Pull request closed without merging**